codesearch/index/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Files generated by failed tests
+__debug_*
```

Comment:
we have one at the root of our repo you could add this to unless there's a reason to have it here

(also needs a newline)

Reply:
moved it up to the root .gitignore - I'll probably ensure the tests clean up after themselves and remove this in the furture